### PR TITLE
Add curl commands to collections page

### PR DESCRIPTION
**Pull request by mjang**
Replaces https://github.com/qdrant/landing_page/pull/561

Link to build page:
https://deploy-preview-565--condescending-goldwasser-91acf0.netlify.app/documentation/concepts/collections/

### CODE REVIEW

File: qdrant-landing/content/documentation/concepts/points.md [hunk 1]
```diff
@@ -520,7 +520,7 @@ In this case, it means that points with the same id will be overwritten when re-
 Idempotence property is useful if you use, for example, a message queue that doesn't provide an exactly-ones guarantee.
 Even with such a system, Qdrant ensures data consistency.
 
-*Available as of v0.10.0*
+[*Available as of v0.10.0*](#create-vector-name)
```
Comment (mjang) [inline qdrant-landing/content/documentation/concepts/points.md:523]: I set up an anchor here to support a reference from the collections.md file, line 683

File: qdrant-landing/content/documentation/concepts/collections.md [hunk 2]
```diff
@@ -578,19 +655,45 @@ automatically be rebuilt in the background to match updated parameters.
...
             "on_disk": true
         }
-    },
+    }
 }
 ```
 
 To put vector data on disk for a collection that **does have** named vectors:
 
+Note: To create a vector name, follow the procedure from our [Points](/documentation/concepts/points/#create-vector-name).
```
Comment (mjang) [inline qdrant-landing/content/documentation/concepts/collections.md:683]: I've modified the points.md file in this PR, with the `#create-vector-name` anchor

File: qdrant-landing/content/documentation/concepts/collections.md [hunk 3]
```diff
@@ -578,19 +655,45 @@ automatically be rebuilt in the background to match updated parameters.
...
+```
+
 ```http
 PATCH /collections/{collection_name}
 {
     "vectors": {
         "": {
             "on_disk": true
         }
-    },
```
Comment (mjang) [inline qdrant-landing/content/documentation/concepts/collections.md:588]: syntax error

File: qdrant-landing/content/documentation/concepts/collections.md [hunk 4]
```diff
@@ -30,8 +30,25 @@ These settings can be changed at any time by a corresponding request.
...
 
+Note: If you're running `curl` from the command line, the following commands
+assume you've set up and are running Docker as described in our
+[Quickstart](/documentation/quick-start/). In that case, you're running Docker 
+on `http://localhost:6333`. For convenience, these commands specify collections
+named `test_collection1` through `test_collection4`.
+
 ## Create a collection
 
+```bash
```
[the remark this answers is not in the document]
Comment (mjang) [reply qdrant-landing/content/documentation/concepts/collections.md:41]: I prefer <code>```curl</code>  as well, but I couldn't get it to work. (IIRC, there's a list of supported languages, and `curl` is not on the list.)

I did test each `curl` command.

File: qdrant-landing/content/documentation/concepts/collections.md [hunk 5]
```diff
@@ -30,8 +30,25 @@ These settings can be changed at any time by a corresponding request.
 
 **When should you create multiple collections?** When you have a limited number of users and you need isolation. This approach is flexible, but it may be more costly, since creating numerous collections may result in resource overhead. Also, you need to ensure that they do not affect each other in any way, including performance-wise. 
 
+Note: If you're running `curl` from the command line, the following commands
+assume you've set up and are running Docker as described in our
+[Quickstart](/documentation/quick-start/). In that case, you're running Docker 
+on `http://localhost:6333`. For convenience, these commands specify collections
+named `test_collection1` through `test_collection4`.
+
```
[the remark this answers is not in the document]
Comment (mjang) [reply qdrant-landing/content/documentation/concepts/collections.md:38]: Oh thank you, excellent idea

File: qdrant-landing/content/documentation/concepts/collections.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (mjang) [reply qdrant-landing/content/documentation/concepts/collections.md:41]: > This is a nitpick, but I think I'd like `` ```curl `` better.

I wish I could get `` ```curl `` to work. (I think `curl` is not on the list of allowed "languages" for markdown code samples)

File: qdrant-landing/content/documentation/concepts/collections.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (mjang) [reply qdrant-landing/content/documentation/concepts/collections.md]: Screenshot from 2024-01-31 06-32-41

File: qdrant-landing/content/documentation/concepts/collections.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (timvisee) [reply qdrant-landing/content/documentation/concepts/collections.md:41]: > (IIRC, there's a list of supported languages, and curl is not on the list.)

Didn't know about this. Thanks!

File: qdrant-landing/content/documentation/concepts/collections.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (timvisee) [reply qdrant-landing/content/documentation/concepts/collections.md]: I didn't necessarily meant in a quote block, but I agree that the text is better like this :+1:

**Review by generall — APPROVED** [review]
LGTM

**Review by timvisee — APPROVED** [review]

**Pull request merged**